package.json

```diff
@@ -60,6 +61,7 @@
     "steal-jasmine": "0.0.3",
     "steal-mocha": "0.0.3",
     "steal-qunit": "^0.1.1",
+    "steal-qunit2": "git://github.com/shayaneumar/steal-qunit.git#78b22c0be77ea622a22f3385d9ebaced5f592e48",
```

Comment:
I have created a fork of steal_qunit repo and created a fork of that, named it to steal-qunit2.

This is done just for testing purposes, wanted to verify that code changes done for qunit 2 is working fine with both qunit 1x and qunit 2x.